packages/app/src/components/uiBlocks/TagEntry.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="flex flex-col flex-shrink-0">
+    <div class="relative pb-1/1">
+      <img class="absolute w-full h-full rounded-xl" :src="image" />
+    </div>
+    <div id="TagName" class="h-2/6 text-white font-semibold" :class="[`text-${textSize}`]">
+      {{ name }}
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
```

Comment:
```suggestion
import { defineComponent, ref } from 'vue';
```

compare line 33

Reply:
fixed in 3be7fb18d799aa08914764c9f315d65ff5a59db5